cmd/analytics.go

```diff
@@ -25,7 +25,7 @@ import (
 func Analytics() *cobra.Command {
 	var disable bool
 	cmd := &cobra.Command{
-		Args:  utils.NoArgsAccepted("https://okteto.com/docs/reference/cli/index.html#analytics"),
+		Args:  utils.NoArgsAccepted("https://okteto.com/docs/reference/cli/#analytics/"),
```

Comment:
For links with has, the trailing slash `/` shouldn't go at the end, but before the `#`:

```suggestion
		Args:  utils.NoArgsAccepted("https://okteto.com/docs/reference/cli/#analytics"),
```